_posts/2015-07-15-testautomation-allergy.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Test automation allergy
+author: pekkama
+excerpt: Questioning the snake-oil approach to test automation
+---
+
+I am allergic to test automation!
+---------------------------------
+
+That's correct! I get itchy and my eyes redden every time some talks about test automation. I get coarse throat when I see a test automation coverage report. I get twitches when we justify our lack of skills as testers with 95% line coverage. That's a reaction to poorly executed and poorly justified test automation. I am allergic to s***ty test automation. And I think everyone should be!
```

Comment:
"I get itchy and my eyes redden every time some talks about test automation." some => someone